Manito/Manito/Screens/Detail-Wait/DetailEditViewController.swift

```diff
@@ -251,12 +246,38 @@ class DetailEditViewController: BaseViewController {
         makeActionSheet(actionTitles: actionTitles, actionStyle: actionStyle, actions: actions)
     }
 
-    private func setChangedButton() {
+    private func setupChangedButton() {
         calendarView.changeButtonState = { [weak self] value in
             self?.changeButton.isUserInteractionEnabled = value
             self?.changeButton.setTitleColor(value ? .subBlue : .grey002, for: .normal)
```

Comment:
p3;

```suggestion
            self?.changeButton.isEnabled = value
            self?.changeButton.setTitleColor(.subBlue, for: .normal)
            self?.changeButton.setTitleColor(.grey002, for: .disabled)
```
이렇게 하면 굳이 value에 상관없이 titleColor가 바뀔 거 같네요

Reply:
오호 for: .normal 과 .disabled를 통해 각각 상태에 따라 설정할 수 있군요 !
감사합니다 ^_______^